screwdriver/detect-what-to-build.sh

```diff
@@ -26,7 +26,9 @@ elif [[ -z $FILES ]]; then
 elif ! grep -v -E "(\.h|\.hh|\.hxx|\.c|\.cpp|\.cxx)$" <<< "$FILES" &> /dev/null; then
   SHOULD_BUILD=cpp
 elif ! grep -v -E "(\.java)$" <<< "$FILES" &> /dev/null; then
-  SHOULD_BUILD=java
+    SHOULD_BUILD=java
+elif ! grep -v -E "(\.go)$" <<< "$FILES" &> /dev/null; then
+    SHOULD_BUILD=go
```

Comment:
Indentation doesn't seem to match the other clauses.

Reply:
Thanks. Fixed now.